.travis.yml

```diff
@@ -97,5 +97,12 @@ matrix:
       - "scripts/build_assembly_no_test.sh scalding-core"
       - "scripts/test_typed_tutorials.sh"
 
+    - scala: 2.10.4
+      env: BUILD="test execution tutorials"
+      script:
+      - "scripts/build_assembly_no_test.sh scalding-core"
```

Comment:
do we need to do this? I think just running the assembly on the next target will do.